### ISSUE TITLE
Is $100 for Node Foundation Individual Membership a barrier to participation?

### ISSUE THREAD

**Issue by jfathman** [nodejs/evangelism#265]
> The Evangelism WG tries to bring more people into Node

More people might become Node Foundation Individual Members if it cost less than $100.  If an employer is paying, that is a small amount, but if the expense is out of pocket, $100 could be a barrier to participation, particularly in regions of the world where that is a substantial sum.

I say this having reached into my own pocket last year for the $100 to become an Individual Member, and questioning whether I can continue that on an annual basis.

Is the objective revenue, or membership?  And at a lower cost, with more participation, revenue could actually rise.

At the risk of over complicating things, might there be a $25 membership (the student level) for individual members who certify their membership is not company sponsored?  And then a (vanity) company sponsored individual membership for $100?

**Comment by mikeal** [comment]
> Is $100 for Node Foundation Individual Membership a barrier to participation? 

Yes :smile: 

We'd like to reduce the cost. Being a "dues paying member" is important but the exact cost is at the foundation's discretion. It's a little late for us to make an adjustment before the election but this is on the agenda in 2017.

**Comment by bnb** [comment]
@mikeal Is there anything actionable we can do with this? Make note of it in nodejs/membership, request it be added to the Board agenda, or something else?

**Comment by mikeal** [comment]
We're looking to re-write the whole reg and vote system so we should mark tag this and other related issues as we start talking about what to do about that.

**Comment by bnb** [comment]
@mikeal Tagged with [membership](https://github.com/nodejs/evangelism/labels/membership) 👍

**Comment by williamkapke** [comment]
There certainly needs to be more incentive. Maybe 20% off Node Interactive for Members? Then people make their money back and it promotes NI.

**Comment by bnb** [comment]
Also, having a Foundation Members list on the site would be a good incentive for many that wouldn't normally splurge on it.

**Comment by jfathman** [comment]
Providing a 'Node Foundation Member' sticker would also be great incentive.

**Comment by williamkapke** [comment]
Whoa. @hackygolucky delivers in https://github.com/nodejs/nodejs.org/pull/1133/files!!

![image](https://cloud.githubusercontent.com/assets/739813/22716704/f401ce60-ed4b-11e6-8a7c-9834fde5a24f.png)

**Comment by mikeal** [comment]
One quick note about this. It's 20% off the "regular price" which does not apply to the crazy early bird tickets we put out that are about half the regular ticket price already :)

**Comment by sashashakun** [comment]
Hello, guys!

Just have a little question about Individual Membership, but don't want to create separate issue. I have a Node Foundation Individual Membership, bought it because of my desire to support node community. And this is what I see after payment: 
<img width="759" alt="2017-02-09 23 48 00" src="https://cloud.githubusercontent.com/assets/5776864/22802603/72635fda-ef22-11e6-83e6-dd605bf3665d.png">
I think it's very strange behaviour. May be someone can add some additional information on this page? Something like, I don'k know,  "HOORAY! YOU ARE NOW NODE FOUNDATION INDIVIDUAL MEMBER!!1", or something more usefull)

P.S. One more little question. Is a list of the community members exist anywhere? May on github, or some where else. Thanks.

**Comment by williamkapke** [comment]
@sashashakun See https://github.com/nodejs/membership/issues/36 re: list of members.
tldr; no

We will be working on improving this system this year. It is fraught with problems unfortunately.

**Comment by bnb** [comment]
@williamkapke Has there been any work toward resolution of this? Would like to see how we can proceed with this specific issue if possible.

**Comment by williamkapke** [comment]
@bnb,
If you're asking about the registration platform, @hackygolucky has been putting a lot of attention to fixing/replacing it and is an ongoing task.

If you were asking about the $100 barrier; The first attempt to help with this is the "20% off regular price registration to Node Interactive 2017" benefit and we're seeing a great response.

**Comment by bnb** [comment]
@williamkapke Generally asking about the issue itself - sounds like there has been 💥 

Is there value in keeping this issue open? That's the main reason I wanted to see if we'd had any further progress.
